pynest/examples/Potjans_2014/README.rst

```diff
@@ -1,61 +1,108 @@
-Microcircuit Example
-=====================
+PyNEST Microcircuit
+===================
 
-Description
------------
+This is a PyNEST implementation of the microcircuit model by Potjans and Diesmann [1]_.
+The network model represents four layers of cortex, L2/3, L4, L5, and L6, each consisting of two populations of excitatory and inhibitory neurons.
 
```

Comment:
I would write 'each consisting of a population of excitatory neurons and a population of inhibitory neurons'

Reply:
Done.